.babelrc

```diff
@@ -0,0 +1,10 @@
+{
+  "presets": [
+      "@parcel/babel-preset-env",
+      "@babel/preset-react"
+  ],
+  "plugins": [
+    "@babel/plugin-proposal-class-properties",
+    "@parcel/babel-plugin-transform-runtime"
```

Comment:
I don’t think this is needed with the current browser support, it’s worth a try running without it

Reply:
Looks like you're right.  FWIW, I was just copy/pasting [from here.](https://v2.parceljs.org/languages/babel/)